packages/playground/website/src/lib/site-metadata.ts

```diff
@@ -39,6 +36,8 @@ export interface SiteMetadata {
 	id: string;
 	name: string;
 	logo?: SiteLogo;
+	// TODO: Don't store this in SiteMetadata.
+	isArchived?: boolean;
```

Comment:
Instead of using `isArchived` maybe we could:
- start supporting and updating `whenLastLoaded` each time a site is loaded
- continue recognizing `opfs-temporary` sites that have been loaded in the last 24 hours
- delete opfs-temporary sites that have been dormant for the last 24 hours

Reply:
I pushed some commits to do this.

Reply:
My worry is the use-case where you create a bunch of Playgrounds by clicking examples or just shopping around Blueprints, and then your entire sidebar gets clogged with maybe 20 sites.

The `isArchived` flag was my idea of addressing that – we could have a separate sidebar button such as "recently archived" and then you go back there to recover your site. Even better, we could archive regular OPFS sites instead of deleting them. Everything from the archive would get deleted after 24 hours.

What do you think?

Reply:
Having an archive sounds nice, we could keep the last three temporary aites in the default sidebar and move all oldrr sites into archived.

Reply:
> My worry is the use-case where you create a bunch of Playgrounds by clicking examples or just shopping around Blueprints, and then your entire sidebar gets clogged with maybe 20 sites.
>
> The isArchived flag was my idea of addressing that 

Your explanation makes sense to me, and archiving sounds like a good idea.